BrainPortal/app/models/cluster_task.rb

```diff
@@ -2168,13 +2132,32 @@ def singularity_commands(command_script)
 # by CbrainTask::ClusterTask
 #   #{CbrainTask::ClusterTask.revision_info.to_s}
 
-# CBRAIN internal consistency test
+# CBRAIN internal consistency test 1: must run under proper UID
 if test "$UID" -ne "#{Process.uid}" ; then
   echo "Singularity internal script running with wrong UID (expected UID=#{Process.uid})"
   echo "Runtime IDs: `id`"
   exit 2
 fi
 
+# CBRAIN internal consistency test 2: must have the task workdir mounted inside the container
+if test ! -d #{task_workdir.bash_escape} ; then
+  echo "Missing mount (#{task_workdir.bash_escape}) point inside the container"
```

Comment:
This echo statement will fail if there are double quotes in the task workdir path. It's supposed to be, to be safe:

```
echo "Container missing mount point for task work directory:" #{task_workdir.bash_escape}
```